tests/baselines/reference/newTarget.es5.js

```diff
@@ -69,11 +69,11 @@ function f1() {
     var g = _newTarget;
     var h = function () { return _newTarget; };
 }
-var f2 = function _a() {
-    var _newTarget = this && this instanceof _a ? this.constructor : void 0;
+var f2 = function _b() {

```

Comment:
What changed the names here?

Reply:
`_a` was always used by the name for the static function expression on `A` - now there's no name generation scope for a class body, so it's the same scope as here, thereby bumping these names to the next character.